### DISCUSSION TITLE
New Grenades

### DISCUSSION THREAD

**Discussion by BringBack50v50** [HasangerGames/suroi#399] [Ideas]
In surviv.io , there were strobes and MIRV grenades. Why can’t there be strobes and MIRV grenades in Suroi.io? There were strobes in the game, Suroi15 I think that we should add them.

**Comment by Emeraldm5** [comment]
We should

**Comment by bien-star** [comment]
Please say this on discord not on the github repository 
Also, strobes were never in normal mode in surviv and there's now a special grenade (c4) to replace mirv
if you want to play surviv how it was, just play resurviv

**Reply by ghost** [reply]
First of all, I don't have discord, nor can I access it.
Secondly, just because surviv didn't have something does not mean that suroi shouldn't have it.
Thirdly, the c4 replaces the mines more than it represents the MIRV grenade.
Lastly, resurviv is not secure and it's last update was in 2019. Same goes for survivreloaded

**Reply by Crimson-Haze** [reply]
he is right, that's why I made an airskirke weapon model

**Reply by Hashimk2** [reply]
> he is right, that's why I made an airskirke weapon model

Who, ‘he’?
Bien-star or SurvivWasTheBestGame
